docs_src/security/Ch-APIGateway.md

```diff
@@ -131,21 +133,33 @@ my-app:
   
 ```
 
-The value of `EDGEX_ADD_PROXY_ROUTE` takes a comma-separated list of one or more (at least one) paired additional service name and URL for which to create proxy Kong routes.   The paired specification is given as the following:
+The value of `EDGEX_ADD_PROXY_ROUTE` takes a comma-separated list
+of one or more (at least one) paired additional service name and URL
+for which to create proxy routes.
+The paired specification is given as the following:
 
     <RoutePrefix>.<TargetRouteURL>
 
-where **RoutePrefix** is the name of service which requests to create proxy Kong route and it is case insensitive; it is the docker network hostname of the service that want to add the proxy Kong route in the docker-compose file if running from docker-compose, for example, `myApp` in this case.
+where **RoutePrefix** is a directory path that will be created off of the root
+of the API gateway to route traffic to the target.
 
-**TargetRouteURL** is the full qualified URL for the target service, like `http://myapp:56789` as it is known on on the network
-on which the API gateway is running.  For Docker,
-the hostname should match the hostname specified in the
-`docker-compose` file.
+**TargetRouteURL** is the full qualified URL for the target service,
+like `http://myapp:56789` as it is known on on the network on which the API gateway is running.
+For Docker, the hostname should match the hostname specified in the `docker-compose.yml` file.
 
 So as an example, for a single service, the value of `EDGEX_ADD_PROXY_ROUTE` would be:  "`myApp.http://myapp:56789`".
 
-Once `EDGEX_ADD_PROXY_ROUTE` is configured and composed-up successfully, the proxy route then can be accessed the app's REST API via Kong as `https://localhost:8443/myApp/api/v2/...` in the same way you would access the EdgeX service.
-You will also need an access token obtained using the documentation below.
+In the given example,
+```
+EDGEX_ADD_PROXY_ROUTE: "myApp.http://my-app:56789"
+```
+
+When a request to the API gateway is received,
+such as `GET https://localhost:8443/myApp/api/v2/ping`,
+the API gateway will reissue the request as
+`GET http://myapp:56789/api/v2/ping`.
```

Comment:
Isn't `my-app-service` same as `my-app?

It is hard to follow because these are all similar names. It is also easy to make a mistake while reading or editing the text. I suggest embedding the label inside the example and keeping a small set of them: `service-hostname` (instead of my-app), `service-name` (instead of myApp).

Reply:
I think it would be least confusing if you stick with a a single naming convention for both route base name (service name) and hostname.

Reply:
> Isn't `my-app-service` same as `my-app?
> 
> It is hard to follow because these are all similar names just presented in different naming conventions. It is also easy to make a mistake while reading or editing the text. I suggest embedding the label inside the example and keeping a small set of them: `service-hostname` (instead of my-app), `service-name` (instead of myApp).

This comment was made in flight as I was addressing the previous comments.  Diregarding for now. Please see latest commit.